pkg/kotsutil/kots.go

```diff
@@ -417,6 +418,8 @@ func LoadKotsKindsFromPath(fromDir string) (*KotsKinds, error) {
 					kotsKinds.IdentityConfig = decoded.(*kotsv1beta1.IdentityConfig)
 				case "kots.io/v1beta1, Kind=Installation":
 					kotsKinds.Installation = *decoded.(*kotsv1beta1.Installation)
+				case "kots.io/v1beta1, Kind=HelmChart":
```

Comment:
i wonder why this wasn't here before now. was it because we don't think of helm charts as kots kinds? i think it's fine to add but might wanna double check with someone else